gradle.properties

```diff
@@ -45,6 +45,7 @@ errorproneVersion = 2.3.3
 
 # Docker settings
 dockerRepositoryPrefix = harbor.h2o.ai/opsh2oai/h2oai/
+dockerUsePython = true
```

Comment:
nit: `dockerIncludePython`? wdyt? IMHO `dockerUserPython` means that you will be using python for the build process, as opposed to including it in the actual container image.